### PR TITLE
Add support python 3 for plugins

### PR DESCRIPTION
**Pull request by Abhishek-Dev09**
Previous version of these plugins is not compatible with python 3.And it also not available on pypi. So, i have build wheels for these plugins that compatible with python 2/3.Also these are universal wheels and platform independent.If possible i can upload these on thirdparty repo.But its not,so it is available in my repo.As three months ago lib has been uploaded by JonoYang except these plugins that support python3.

Signed-off-by: Abhishek Kumar abhishek.kasyap09@gmail.com

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -121,7 +121,11 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    python_requires='>=2.7,<4',    
+    dependency_links = ['https://github.com/Abhishek-Dev09/libarchive/raw/master/extractcode-libarchive-3.1.2.tar.gz#egg=extractcode-libarchive-3.1.2',
```
Comment (pombredanne) [inline setup.py]: I cannot merge things that would work with a dependency link that is off your repository. If you modified somehow these plugins... why not make this a PR? the code is under `/plugins`

File: setup.py [hunk 2]
```diff
@@ -121,7 +121,11 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    python_requires='>=2.7,<4',
```
Comment (pombredanne) [inline setup.py]: This is a small but important change... so you should make this its own commit

File: setup.py (hunk 1 above)
Comment (Abhishek-Dev09) [reply setup.py]: I do not have push access to your repository.Thats why i have uploaded on my repo.
No need to worry i have built from /plugins only.
